cafebabel/static/img/profile.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="168.444px" height="168.186px" viewBox="0 0 168.444 168.186" enable-background="new 0 0 168.444 168.186"

```

Comment:
I think you can even change the width/height directly here (don't forget to change the `viewBox` and `enable-background` accordingly 😉 )

Reply:
ok